src/exojax/spec/opacalc.py

```diff
@@ -333,9 +333,9 @@ def xsvector(self, T, P):
         nsigmaD = normalized_doppler_sigma(T, self.mdb.molmass, R)
 
         if self.mdb.dbtype == "hitran":
-            qt = self.mdb.qr_interp(self.mdb.isotope, T)
+            qt = self.mdb.qr_interp(self.mdb.isotope, T, Tref_original)
         elif self.mdb.dbtype == "exomol":
-            qt = self.mdb.qr_interp(T)
+            qt = self.mdb.qr_interp(T, Tref_original)
```

Comment:
`qt = self.mdb.qr_interp(T, Tref_original)` should be `qt = self.mdb.qr_interp(T, self.Tref_broadening)`